crates/bevy_ecs/src/system/system.rs

```diff
@@ -59,6 +60,10 @@ pub trait System: Send + Sync + 'static {
     fn default_labels(&self) -> Vec<SystemLabelId> {
         Vec::new()
     }
+    /// Allows users to get the system's last change tick.
```

Comment:
"Users" in this context is implicit. I think this should just say something like "Gets the system's last change tick". Same for the set method.